docs/02-app/02-api-reference/04-functions/use-router.mdx

```diff
@@ -41,8 +41,8 @@ export default function Page() {
 
 ## `useRouter()`
 
-- `router.push(href: string)`: Perform a client-side navigation to the provided route. Adds a new entry into the [browser’s history](https://developer.mozilla.org/en-US/docs/Web/API/History_API) stack.
-- `router.replace(href: string)`: Perform a client-side navigation to the provided route without adding a new entry into the [browser’s history stack](https://developer.mozilla.org/en-US/docs/Web/API/History_API).
+- `router.push(href: string, options?: { scroll: false })`: Perform a client-side navigation to the provided route. Adds a new entry into the [browser’s history](https://developer.mozilla.org/en-US/docs/Web/API/History_API) stack. You can optionally set `scroll: false` when you don't need to scroll back to top again when the URL changes (For instance hash query changes.)
```

Comment:
@delbaoliveira this formatting looks a bit off to me because we don't want people to always copy `scroll` when looking at the docs. Is there a bette place to explain that?